### ISSUE TITLE
0.0.55 release missing some fixes

### ISSUE THREAD

**Issue by jmatsushita** [automerge/automerge-repo#84]
Hi there,

Awesome to see the recent fixes! However looking at the `dist` folder in the `0.0.55` release here https://www.npmjs.com/package/automerge-repo?activeTab=code I can see that [some fixes](https://github.com/automerge/automerge-repo/pull/81) didn't make it.
```js
import { withTimeout, TimeoutError } from "./helpers/withTimeout";
```

Cheers!

Jun

**Comment by pvh** [comment]
I'll ship an 0.0.56. Thanks.

**Comment by acurrieclark** [comment]
This is fixed in `0.0.56`. Thanks for the report @jmatsushita
